cli/src/common.ts

```diff
@@ -168,6 +170,7 @@ export async function runPlatformHook(
       cwd: platformDir,
       env: {
         INIT_CWD: platformDir,
+        CAP_CONFIG: config,
```

Comment:
There are a couple things we will need to address here:
- I'm not sure how `config` is being serialized into an environment variable here, we'll need to check.
- The CLI has "lazy values" now (https://github.com/ionic-team/capacitor/pull/4007) and any place we want the **full** config we'll need to use that `deepAwait` helper.
- But... I'm not sure we want to expose the full config here. Maybe just `config.app.extConfig` and a few key directories? `config.app.rootDir`?